### PR TITLE
Create ui-c8y-1018-503-119-translation-added-from-public-options-will-now-have-higher-priority-than.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by ensu

Ticket: [MTM-60268](https://cumulocity.atlassian.net/browse/MTM-60268)
Original PR: [6986](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6986)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1018-503-119-translation-added-from-public-options-will-now-have-higher-priority-than.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Translation added from public options will now have higher priority than plugin po files (#6986)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1018-503-119-translation-added-from-public-options-will-now-have-higher-priority-than.md]: The Claude AI suggested to use the following title:
```suggestion
title: Public option translations now have higher priority than plugin po files
```

File: content/change-logs/application-enablement/ui-c8y-1018-503-119-translation-added-from-public-options-will-now-have-higher-priority-than.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60268
+version: 1018.503.119
+---
+Translation added from public options will now have higher priority than plugin po files (#6986)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1018-503-119-translation-added-from-public-options-will-now-have-higher-priority-than.md]: The Claude AI suggested to use the following body:
```suggestion
Previously, translations from plugin po files took precedence over translations added via public options, which could lead to inconsistencies. With this change, translations added via public options will now have a higher priority than translations from plugin po files. This ensures that custom translations specified in the public options are correctly applied and displayed in the user interface, providing a more consistent and predictable translation behavior for users.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by BeateRixen** [comment]
@eniosultan Please check this PR

**Review by eniosultan — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**